lib/manager/components/FeedSourceTableRow.js

```diff
@@ -124,46 +120,72 @@ function relativeDuration (time: any) {
   )
 }
 
+//
+/**
+ * This method is extracted variable generation which had to be split out
```

Comment:
Sentence does not make sense to me.

Reply:
Have reworded it and I hope it makes more sense now!